address/script.go

```diff
@@ -0,0 +1,350 @@
+package address
+
```

Comment:
same for this file.

Reply:
ExtractScriptAddrs method was slightly modified ExtractPkScriptAddrs which at one point I needed(to check if fedpeg script is multisig etc) but later i saw that i dont need to use it, so now i remove it.